### PR TITLE
feat(infra): add notice component

### PR DESCRIPTION
**Pull request by tomerye**
### Description

adding notice component

image

### Todos

- [ ] Updated and added unit tests
- [ ] Added Cypress tests

### CODE REVIEW

File: src/components/FbxNotice/FbxNotice.vue [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+<script>
+export default {
+  name: 'FbxNotice'
+}
+</script>
+
+<style lang="scss" scoped>
+  .info-message {
+        background-color: #deebf9;
```
[the remark this answers is not in the document]
Comment (tomerye) [reply src/components/FbxNotice/FbxNotice.vue]: thanks!

File: src/components/FbxNotice/FbxNotice.stories.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import { withInfo } from 'storybook-addon-vue-info'
+
+import FbxNotice from './FbxNotice.vue'
+import summary from './FbxNotice.md'
+
+const stories = storiesOf('Components/Notice', module)
+
+stories.add('default', withInfo({ summary })(() => ({
+  components: { FbxNotice },
+  template: `<fbx-notice><div style="padding: 16px">Blaaaa Blaaa</div></fbx-notice>`
```
Comment (kindofone) [inline src/components/FbxNotice/FbxNotice.stories.js]: Could be more professional to add a real text or lorem ipsum at least.

File: src/components/FbxNotice/FbxNotice.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# FbxNotice
+display the content centered with blue(currently) color as background
```
Comment (kindofone) [inline src/components/FbxNotice/FbxNotice.md]:
```suggestion
Display centered content on a blue strip.
```

**Pull request merged**